### PR TITLE
archive(8.2) isolation: pt 2, reviewable steps

### PR DESCRIPTION
**Pull request by pepopowitz**
Note: this PR currently targets `unsupported/8.2`. These changes will not make it into`main`. 

## Description

Part of #4527. 

This PR includes the reviewable steps for isolating an 8.2 website. After this merges, I'll publish a standalone 8.2 release.

Most of the steps in this PR were automated, but the last couple are manual:

### Progress

- [x] update version archival process to account for infrastructure changes (the changes from #4624, which were not in main when I created `unsupported/8.2`.)
- [x] exclude all other versions from the main version file
- [x] update version numbers for isolation
- [x] exclude all other versions from the optimize version file
- [x] prevent unsupported site from being crawled
- [x] fix Dockerfile for local debugging
- [x] update theme components
- [x] update CI workflows
- [x] update docusaurus config
- [x] configure Optimize docs instance
- [x] ------- above this was handled automatically; below is manual ---------
- [x] fix incorrect links
- [ ] fix htaccess rules

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

- [x] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process).

### CODE REVIEW

File: versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ To maintain a single source of truth, avoid multiple Web Modeler instances for d
...
+Pipeline-driven deployment can be executed for a single file or an entire project. A separate system of record, maintained outside Web Modeler, can handle finer-grained dependency management. Fetch the full project for a file using the `GET /api/beta/files/:id` endpoint to acquire the project's `projectId`. Subsequently, use the `POST /api/beta/files/search` offered starting from Web Modeler 8.3 on to retrieve all project files, see [the 8.3 documentation](https://docs.camunda.io/docs/8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd/#automate-deployment-of-linked-resourcesdependencies).
```
[the remark this answers is not in the document]
Comment (pepopowitz) [reply versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md:135]: Intentionally hardcoding to production, to cross versions.

File: versioned_docs/version-8.2/apis-tools/web-modeler-api/index.md [hunk 2]
```diff
@@ -144,4 +144,4 @@ Internally, the ids are what matters. You can rename files or move files between
 ### How do I migrate from the `beta` API to the `v1` API? {#migrating-from-beta-to-v1}
 
 Web Modeler's stable `v1` API is offered starting from Web Modeler 8.3.
-For migration hints, see the [Web Modeler 8.3 API documentation](/versioned_docs/version-8.3/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
+For migration hints, see the [Web Modeler 8.3 API documentation](https://docs.camunda.io/docs/8.3/apis-tools/web-modeler-api/#migrating-from-beta-to-v1).
```
[the remark this answers is not in the document]
Comment (pepopowitz) [reply versioned_docs/version-8.2/apis-tools/web-modeler-api/index.md:147]: Intentionally hardcoding to production, to cross versions.

File: versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md [hunk 3]
```diff
@@ -82,15 +82,15 @@ Synchronize files between Web Modeler and version control systems (VCS) and vice
...
 
 Real-time synchronization isn't always what you need. Consider Web Modeler as a local repository, and update your remote repository only after files are committed and pushed. This aligns with the concept of [milestones](/components/modeler/web-modeler/milestones.md).
 
 #### Listening to milestone creation
 
 A milestone reflects a state of a file in Web Modeler with a certain level of qualification, such as being ready for deployment. You can use this property to trigger deployments when a certain milestone is created.
 
-Starting from 8.3, Web Modeler offers a polling approach to listen to milestone creation, see [the 8.3 documentation](/versioned_docs/version-8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md#listening-to-milestone-creation).
+Starting from 8.3, Web Modeler offers a polling approach to listen to milestone creation, see [the 8.3 documentation](https://docs.camunda.io/docs/8.3/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd/#listening-to-milestone-creation).
```
[the remark this answers is not in the document]
Comment (pepopowitz) [reply versioned_docs/version-8.2/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md:93]: Intentionally hardcoding to production, to cross versions.

File: src/pages/index.js [hunk 4]
```diff
@@ -134,45 +134,6 @@ function Home() {
             </div>
           </section>
         )}
-        <div className={clsx("hero hero--secondary", styles.heroBanner)}>
```
Comment (pepopowitz) [inline src/pages/index.js:137]: Removing this entire section, because only 1 of 3 links in here applies to this version.

**Review by akeller — APPROVED** [review]
This looks good, however, I just skimmed the link changes.

**Pull request merged**